src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -597,21 +597,39 @@ private AdsDataProvider() {
   }
 
   @DataProvider
-  public static Object[][] fliteTagBroken() {
+  public static Object[][] fliteTagBrokenOasis() {
     return new Object[][]{
         {"adtest", "SyntheticTests/FliteTagBrokenWidth",
-         "Invalid width of the flite unit was passed. Make sure you provide width parameter with numeric value."},
+         "Invalid width of the flite unit was passed. Make sure you provide width parameter with numeric value.", "#mw-content-text .error"},
         {"adtest", "SyntheticTests/FliteTagBrokenHeight",
-         "Invalid height of the flite unit was passed. Make sure you provide height parameter with numeric value."},
+         "Invalid height of the flite unit was passed. Make sure you provide height parameter with numeric value.", "#mw-content-text .error"},
         {"adtest", "SyntheticTests/FliteTagBrokenTag",
-         "Invalid guid parameter was passed. Provide valid guid or remove this tag from article's content."}};
+         "Invalid guid parameter was passed. Provide valid guid or remove this tag from article's content.", "#mw-content-text .error"}};
   }
 
   @DataProvider
-  public static Object[][] fliteTag() {
+  public static Object[][] fliteTagBrokenMercury() {
     return new Object[][]{
-        {"adtest", "SyntheticTests/FliteTag"},
-        {"adtest", "SyntheticTests/FliteTagModifiedTag"}};
+        {"adtest", "SyntheticTests/FliteTagBrokenWidth",
+         "Invalid width of the flite unit was passed. Make sure you provide width parameter with numeric value.", ".ember-view.article-content.mw-content .error"},
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal ".ember-view.article-content.mw-content .error" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)